Straltsou.TMS0209.sln

```diff
@@ -3,24 +3,30 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29911.84
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleCalculator", "SimpleCalculator\SimpleCalculator.csproj", "{AE4D92B3-F3D0-4803-9FF2-72506EE4A733}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HelloWorldProj", "src\HelloWorld\HelloWorldProj.csproj", "{850F69B5-2425-4CE3-BEE6-40E08BB08CB3}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleCalculator", "src\SimpleCalculator\SimpleCalculator.csproj", "{F8777F82-9547-44A0-A982-9681D04FCFD4}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TaskHandler", "..\..\TMS Test\Tasks Draft\Example1\TaskHandler.csproj", "{811E24BC-480E-4383-A5CF-D47F7BB16720}"
```

Comment:
Bad path to project. Change it to: "TaskHandler", "src\TaskHandler\TaskHandler.csproj"